### DISCUSSION TITLE
We're sorry, but something went wrong. If you are the application owner check the logs for more information.

### DISCUSSION THREAD

**Discussion by kamnev999** [postalserver/postal#3145] [Help with using Postal]
Hello.
I ran into a problem.

I created the end point of the route., created the route, after that I deleted the end point of the route.

Now when switching to domen.com/org/test/servers/1/routes

I get an error :
I'm sorry, but something went wrong.
If you are the owner of the app, check the logs for more information.

I want to try to delete the route, but I don't know what the variable is called.

domen.com/org/test/servers/1/routes
/*****/edit 

I didn't find any information in the logs, most likely I'm not displaying the logs correctly.

**Comment by willpower232** [comment]
Have you tried `postal logs web`? That would give you a clue,

I imagine you would either have to fix it from the terminal somehow or connect to your Postal database server with a GUI to remove the half deleted row.
